sale_order_import/tests/test_parse_validate.py

```diff
@@ -29,23 +29,14 @@ def test_onchange_validation_not_supported(self):
         # Test it gets called (cannot do it w/ Form)
         mock_file_msg = mock.patch.object(type(self.wiz_model), "_unsupported_file_msg")
         with mock_file_msg as mocked:
-            with Form(self.wiz_model) as form:
-                form.order_filename = "test.omg"
+            with Form(
+                self.wiz_model.with_context(default_order_filename="test.omg")
+            ) as form:
                 form.order_file = "00100000"
                 self.assertFalse(form.csv_import)
                 self.assertFalse(form.doc_type)
                 mocked.assert_called()
 
-    def test_onchange_validation_csv(self):
-        csv_data = base64.b64encode(b"id,name\n,1,Foo")
-
-        with Form(self.wiz_model) as form:
-            form.partner_id = self.partner  # required by the view if CSV is set
-            form.order_filename = "test.csv"
-            form.order_file = csv_data
-            self.assertTrue(form.csv_import)
-            self.assertFalse(form.doc_type)
-
```

Comment:
I have only removed test and not removed code related to CSV in wizard
because maybe in the future, `sale_order_import_csv` has been migrated and inherited on them
I think so